web/web.go

```diff
@@ -146,9 +146,13 @@ func New(st local.Storage, qe *promql.Engine, rm *rules.Manager, status *Prometh
 	instrf := prometheus.InstrumentHandlerFunc
 	instrh := prometheus.InstrumentHandler
 
-	router.Get("/", instrf("status", h.status))
+	router.Get("/", instrf("graph", h.graph))
```

Comment:
One of the things we need to do is remove the clutter from the current start page, a full list of targets, rules and configs is too much (build information is fine, though could be more compact). Each of those should have their own page, and only a summary on the status page - e.g. only a row per job/scrape config.